README.md

```diff
@@ -70,25 +70,44 @@ smoochCore.webhooks.get(id).then(function(response) {
 ```
 
 ## API
-This is an overview of what the library has to offer. A better documentation will be available soon.
-
-| Module    | Method  | Endpoint |
-|--------   |-------- |-------------|
-| appUsers  | init    | [POST `/v1/init`](http://docs.smooch.io/rest/#init-beta)       |
-|           | get     | [GET `/v1/appusers/:id`](http://docs.smooch.io/rest/#get-app-user)         |
-|           | update  | [PUT `/v1/appusers/:id`](http://docs.smooch.io/rest/#update-app-user)         |
-|           | trackEvent  | [POST `/v1/appusers/:id/events`](http://docs.smooch.io/rest/#track-event)         |
-|           | getMessages | [GET `/v1/appusers/:id/messages`](http://docs.smooch.io/rest/#get-conversation) |
-|           | sendMessage | [POST `/v1/appusers/:id/messages`](http://docs.smooch.io/rest/#post-message) |
-|           | uploadImage | [POST `/v1/appusers/:id/images`](http://docs.smooch.io/rest/#upload-image) |
-|           | linkChannel | [POST `/v1/appusers/:id/channels`](http://docs.smooch.io/rest/#link-app-user-to-channel) |
-|           | unlinkChannel | [POST `/v1/appusers/:id/channels/:channel`](http://docs.smooch.io/rest/#unlink-app-user-from-channel) |
-|           | typingActivity | [POST `/v1/appusers/:id/conversation/:activity`](http://docs.smooch.io/rest/#typing-activity) |
-| webhooks | list | [GET `/v1/webhooks`](http://docs.smooch.io/rest/#list-webhook) |
-|  | create | [POST `/v1/webhooks`](http://docs.smooch.io/rest/#create-webhook) |
-|  | get | [GET `/v1/webhooks/:id`](http://docs.smooch.io/rest/#get-webhook) |
-|  | update | [PUT `/v1/webhooks/:id`](http://docs.smooch.io/rest/#update-webhook) |
-|  | delete | [DELETE `/v1/webhooks/:id`](http://docs.smooch.io/rest/#delete-webhook) |
-| menu      | get      | [GET `/v1/menu`](http://docs.smooch.io/rest#persistent-menus)       |
-|           | configure| [PUT `/v1/menu`](http://docs.smooch.io/rest#persistent-menus)   |
-|           | remove   | [DELETE `/v1/menu`](http://docs.smooch.io/rest#persistent-menus)|
+
+Below is a list of methods included in Smooch Core. For comprehensive documentation of Smooch Core and it's methods see Smooch's [REST API docs](https://docs.smooch.io/rest/).
```

Comment:
I would add. If a method is missing please file an [Issue](https://github.com/smooch/smooch-core-js/issues) or better yet make a [PR](https://github.com/smooch/smooch-core-js/pulls) 😍